src/xbot/common/math/PIDManager.java

```diff
@@ -14,17 +14,30 @@
     private DoubleProperty maxOutput;
     private DoubleProperty minOutput;
     private BooleanProperty isEnabled;
-
-    public PIDManager(String functionName, XPropertyManager propMan, double defaultP, double defaultI, double defaultD,
-            double defaultMaxOutput, double defaultMinOutput) {
-        super(functionName, propMan, defaultP, defaultI, defaultD, 0);
+    
+    public PIDManager(String functionName, XPropertyManager propMan, double defaultP, double defaultI, double defaultD, double defaultF,
```

Comment:
It's hard to tell from the github view, but it was already broken up into a few line segments, so checkstyle was not being hit. It's since been put into individual lines per argument